packages/headless-react/src/ssr-commerce/index.ts

```diff
@@ -0,0 +1,4 @@
+export {defineCommerceEngine} from './commerce-engine.js';
+export type {ReactCommerceEngineDefinition} from './commerce-engine.js';
+export {MissingEngineProviderError} from '../errors.js';
```

Comment:
I see we do the same for headless-react/ssr but why are we exporting this error ? No users will ever want to import this and throw it.

Reply:
Right, I didn't think about catching it 😄